templates/docs/document.html

```diff
@@ -0,0 +1,74 @@
+{% extends "base_index.html" %}
+
+{% block page_class %}docs{% endblock %}
+{% block content_class %}l-docs-wrapper{% endblock %}
+{% block page_title %}| {{ document.title }} {% endblock %}
+
+{% block content %}
+  <section id="search-docs" class="p-strip--image is-shallow" style="background-image: url('https://assets.ubuntu.com/v1/e54487e2-maas-docs-suru.png')">
+    <div class="row">
+      <form class="p-search-box u-no-margin--bottom" style="box-shadow: none" action="/docs/search">
```

Comment:
on links I try to use `url_for('name of the function')` to not be dependent of the urls in case there is a change. 

`url_for` will get the defined url in the flask routes. https://flask.palletsprojects.com/en/1.1.x/api/?highlight=url_for#flask.url_for

Reply:
I'm aware of `url_for`. I prefer not to use it. The reason is that to me its usage us about where you want your locus of control to be. If you expect the important people trying to understand you app to be reading Python and want to avoid reading HTML, then they would rather be able to find and change all the URLs in `app.py` or `views.py` or `urls.py` or whatever.

However, if they are more likely to be reading HTML, then having to work out how `url_for` generates a URL and where its stored is hard for them.

I think it's quite clear in our case that it's much more useful for the majority of the team for the HTML to be direct than for everything to be controlled through Python.